frontend/libs/health-department/domain/src/lib/resolvers/tracked-case-diary-entries.resolver.ts

```diff
@@ -0,0 +1,20 @@
+import {Observable, of} from 'rxjs';
+import {Injectable} from '@angular/core';
+import {ActivatedRouteSnapshot, Resolve} from '@angular/router';
+import {HealthDepartmentService} from "../data-access/health-department.service";
+import {TrackedCaseDiaryEntryDto} from "@qro/health-department/domain";
+
+@Injectable()
+export class TrackedCaseDiaryEntriesResolver implements Resolve<TrackedCaseDiaryEntryDto[]> {
```

Comment:
Hinweis: In dem PR von Sebastian werden die client daten für das HD mit ngrx gecacht. Das muss hier dann noch nachgezogen werden

Reply:
Die Anpassungen von Sebastian sind inzwischen in develop. Du kannst deinen Code jetzt entsprechend anpassen :)